src/helpers/augmentedReality/multipleSceneIndexGenerator.ts

```diff
@@ -80,6 +80,14 @@ export const multipleSceneIndexGenerator = ({
     }
 
     localUris = scenes?.[modelIndex]?.localUris;
+
+    if (modelIndex !== 0) {
```

Comment:
we want this for > 0, and not for negative values, or am i wrong? comparison with > 0 would read better here i guess?

Reply:
That's right. Index will never be a negative number. 👍